### ISSUE TITLE
error on startup

### ISSUE THREAD

**Issue by ian-paterson** [dbcli/mycli#196]
When I try to open the program using:
```
mycli -h 192.168.0.1 -u username -psecret
```

I'm getting this error
```
Traceback (most recent call last):
  File "/usr/local/Cellar/mycli/1.5.1/libexec/bin/mycli", line 9, in <module>
    load_entry_point('mycli==1.5.1', 'console_scripts', 'mycli')()
  File "/usr/local/Cellar/mycli/1.5.1/libexec/vendor/lib/python2.7/site-packages/click/core.py", line 700, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/Cellar/mycli/1.5.1/libexec/vendor/lib/python2.7/site-packages/click/core.py", line 680, in main
    rv = self.invoke(ctx)
  File "/usr/local/Cellar/mycli/1.5.1/libexec/vendor/lib/python2.7/site-packages/click/core.py", line 873, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/Cellar/mycli/1.5.1/libexec/vendor/lib/python2.7/site-packages/click/core.py", line 508, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/Cellar/mycli/1.5.1/libexec/lib/python2.7/site-packages/mycli/main.py", line 639, in cli
    mycli.connect(database, user, password, host, port, socket)
  File "/usr/local/Cellar/mycli/1.5.1/libexec/lib/python2.7/site-packages/mycli/main.py", line 292, in connect
    port = int(port or cnf['port']) or 3306
TypeError: int() argument must be a string or a number, not 'NoneType'
```

**Comment by amjith** [comment]
Yup. That is a bug. Sorry about that. I'll release a fix for it asap.

In the meantime you can work around it by providing a port number in command line 

```
mycli -h 192.168.0.1 -u username -psecret -P 3306
```

**Comment by ian-paterson** [comment]
@amjith Thanks for the quick fix!

**Comment by amjith** [comment]
This is now fixed in version 1.5.2. Please upgrade: 

```
pip install -U mycli
```
